sublaunching/sublaunching_tutorial/README.md

```diff
@@ -0,0 +1,52 @@
+# Running SLiM in arrays, from Python, from R, and on a computing cluster
+
+#### Author: Samuel Champer
+
+You may find yourself in the situation that you have a SLiM simulation with several defined parameters, and you want to try varying these parameters through a plausable range in order to determine how the model will perform in various configurations.
+
+Naturally, you can just vary the parameters by hand and record the results. However, that wouldn't be very fun once you have more than two or three parameters that you want to vary. Furthermore, running SLiM many times sequentially takes a lot longer than doing it in parallel!
+
+In this situation, there are several options. This repository is designed to demostrate the basics of several methods of solving this problem!
+
+Note: the scripts in this repository all assume that SLiM is in your PATH environment variable. To check if it is, open a console and type "slim". If the SLiM version info and command line help pops up, you're good to go. Otherwise, you'll need to add it to PATH. If you don't know how, the method for doing so varies between operating systems - search for a tutorial that is right for yours.
+
+Note 2: The scripts in this repository require Python version 3.6 or above.
+
+## 1 - Preparing your SLiM file for command line configuration
+
+Just as you can use the defineConstant() function to... well... define constants within SLiM, you can use the command line "-d" argument to do so as well.
+
+For example invoking: ``slim -d X=0.0 -d Y=1.0 filename.slim`` will run SLiM with X in the namespace as 0 and Y as 1. This can be used in conjunction with other tools to run many SLiM simulations while varying parameters.
+
+However, you'll need to make sure not to define the same constants twice! Doing so results in an error. To define a constant "X" to have a default value within SLiM in such a way that it can also be defined via the command line without causing an error, use the following code:
+```
+if (!exists("X"))
+    defineConstant(X, 1.0);
+```
+However, if you plan to use lots of configurable constants, this may start to take up a lot of space, and a small helper function can clean this up. See the "minimal_gene_drive.slim" file for an example of how this is done.
+
+## 2 - Using bash to run SLiM
+
+Once your SLiM file is prepared, running it with varrying parameters using bash is easy. Just a simple list of the command line strings you want to run is sufficient, though you may also want to pipe the output of each simulation to its own file via "``>``" (e.g. "``command > output.txt``"). You can also run jobs in parallel very easily.
```

Comment:
*varying

Reply:
Fixed... I should really use a text editor with a spell checker, eh?